### DISCUSSION TITLE
Qt6?

### DISCUSSION THREAD

**Discussion by rcalixte** [akiyosi/goneovim#558] [General]
Have you thought about porting this application to Qt6? Support for Qt5 will be phased out by the end of 2025. There are newly released [Qt bindings for Go](https://github.com/mappu/miqt) with support for Qt5 and Qt6 and the interface is mostly similar to the previous Qt bindings. I [recently converted](https://github.com/rcalixte/jqview/commit/5e16f1f44eebd4c70f36c3e09a427c95ffb106ac) a much smaller application to Qt6 without much fuss. (I did have one issue with a goroutine thread panicking because of an issue with the window widget pointer but I was able to work around it.) The author @mappu has also been extremely responsive. It would be great to reestablish a community around Go and Qt development!

**Comment by akiyosi** [comment]
@rcalixte 
Thank you for bringing this up. 
I was aware that Qt5 will be gradually phased out by 2025, so this was on my mind as something to address. I wasn't familiar with the new Qt bindings for Go, but given our project’s direction, I'd like to work toward supporting Qt6. 
I’ll take a closer look at these new bindings you mentioned. Thanks again for the heads-up and for sharing your experience!
